src/Sarif/Schemata/Sarif.schema.json

```diff
@@ -586,7 +586,12 @@
         {
           "description": "A dictionary each of whose keys is a URI and each of whose values is an array of fileReference objects representing the location of a single file target scanned during the run.",
           "type": "object",
-          "default": {}
+          "additionalProperties": {
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/fileReference"
+            }
+          }
```

Comment:
This is where `fileInfo` becomes a dictionary from `string` to `IList<FileReference>`. As we've discussed, we might choose to represent this with `patternProperties` rather than `additionalProperties`. But the change to Json.Schema to support `patternProperties` is small compared to the amount of work that went into making Json.Schema.ToDotNet generate the code to support arbitrary dictionaries.